TriblerGUI/dialogs/createtorrentdialog.py

```diff
@@ -30,11 +30,7 @@ def __init__(self, parent):
 
 class CreateTorrentDialog(DialogContainer):
 
-    button_clicked = pyqtSignal(int)
-    received_metainfo = pyqtSignal(dict)
-
-    signal_torrent_created = pyqtSignal(dict)
-    signal_torrent_added_to_channel = pyqtSignal(dict)
+    signal_create_torrent_updates = pyqtSignal(dict)
```

Comment:
Please, rename the signal to something more specific, like `create_torrent_tick` or `create_torrent_finished`, etc.